bin/stop

```diff
@@ -1,5 +1,4 @@
 #!/bin/sh
 set -exu
 
-CONTAINER_NAME='hmtoken.ganache'
-docker rm -f $CONTAINER_NAME || true
```

Comment:
You need to docker-compose rm as well.

Reply:
This runs `docker-compose down` now. `docker-compose rm` gives: `No stopped containers`

Reply:
Yeah down (especially with a `-v` [volumes]) will throw away the containers and networks in this docker-compose